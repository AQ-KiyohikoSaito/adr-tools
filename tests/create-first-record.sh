adr new The First Decision
head -4 doc/adr/0001-the-first-decision.md
