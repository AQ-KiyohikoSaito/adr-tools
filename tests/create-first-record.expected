+ adr new The First Decision
doc/adr/0001-the-first-decision.md
+ cat doc/adr/0001-the-first-decision.md
# 1. The First Decision

Date: 12/01/1992

## Status

Accepted

## Context

Context here...

## Decision

Decision here...

## Consequences

Consequences here...
